code/modules/power/apc.dm

```diff
@@ -1363,6 +1374,23 @@
 			charging = APC_NOT_CHARGING
 			chargecount = 0
 
+		if(excess >= APC_ARC_LOWERLIMIT && !shock_proof) // SKYRAT EDIT ADD - APC Arcing
+			var/shock_chance = 5 // 5%
+			if(excess >= APC_ARC_UPPERLIMIT)
+				shock_chance = 15
+			else if(excess >= APC_ARC_MEDIUMLIMIT)
+				shock_chance = 10
+			if(prob(shock_chance))
+				var/list/shock_mobs = list()
+				for(var/creature in view(get_turf(src), 5)) //We only want to shock a single random mob in range, not every one.
+					if(isliving(creature))
+						shock_mobs += creature
+				if(shock_mobs.len)
+					var/mob/living/living_target = pick(shock_mobs)
+					living_target.electrocute_act(rand(5, 25), "electrical arc")
+					playsound(get_turf(living_target), 'sound/machines/defib_zap.ogg', 75, TRUE) //Man the defib zap sound is cool and also I couldn't find the proper sound.
```

Comment:
I'd prefer this was an electrical arcing sound https://youtu.be/_2LpCdhuOyQ?t=48

Reply:
I found the actual arcing sound and it uses that now